docker-compose.yml

```diff
@@ -69,13 +69,11 @@ services:
 
   backend:
     image: dlrriesgos/backend
-    ports:
-      - ${backendPort}:${backendPort}
     volumes:
       - logs:/backend/data/logs
       - store:/backend/data/store
     environment:
-      port: ${backendPort}
```

Comment:
Do we still need to have the backendPort as env variable documentated in the readme.md?

Reply:
`backendPort`removed from .env!